src/game

```diff
@@ -8,7 +8,17 @@ import typer
 import moreorless
 
 app = typer.Typer()
-SCOREBOARD_PATH = "scoreboard.json"
+SCOREBOARD_PATH = "src/scoreboard.json"
+
+
+@app.command()
+def erase_scoreboard(scoreboard_path: pathlib.Path = SCOREBOARD_PATH):
+    """
+    More or less Erase Scoreboard
+    """
+    moreorless.erase_player_score(player_name=str,
+                                  player_score=int,
```

Comment:
The objective is to erase all the scoreboard, does not need for player_name or score ?

Reply:
It's updated now by removing 'player_name and player_score' parameters